src/commands/publish/mod.rs

```diff
@@ -20,13 +20,23 @@ use crate::settings::global_user::GlobalUser;
 use crate::settings::target::Target;
 use crate::terminal::{emoji, message};
 
-pub fn publish(user: &GlobalUser, target: &Target) -> Result<(), failure::Error> {
+pub fn publish(
+    user: &GlobalUser,
+    target: &Target,
+    no_worker: bool,
```

Comment:
can we change these booleans from double negatives? `!no_worker` is confusing. perhaps `push_worker` and `push_bucket` or something like that?

Reply:
for the publish command sure, but these are negation flags, so their purpose is to suppress standard behavior, so the flag itself is a negative. we can flip it to pass to publish, though.